### PR TITLE
Custom cg path to get rid of more cg alerts in various oryx builds

### PR DESCRIPTION
**Pull request by arroyc**
- [x] The purpose of this PR is explained in this message or in an issue. 
- [ ] Tests are included and/or updated for code changes.
- [x] Proper license headers are included in each file.

### CODE REVIEW

File: vsts/pipelines/ci-detector.yml [hunk 1]
```diff
@@ -15,6 +15,11 @@ stages:
       variables:
         skipComponentGovernanceDetection: true
       steps:
+      - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+        displayName: 'Component Detection - OSS Compliance'
+        inputs:
+          ignoreDirectories: '$(Build.SourcesDirectory)/tests'
```
Comment (gpcastro) [inline vsts/pipelines/ci-detector.yml:21]: how about we add only the sample apps directory?

Comment (arroyc) [reply vsts/pipelines/ci-detector.yml:21]: we can try that as well

**Review by william-msft — APPROVED** [review]

File: vsts/pipelines/autoUpdater.yml [hunk 2]
```diff
@@ -10,6 +10,11 @@ steps:
   inputs:
     version: 3.1.x
 
+- task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+  displayName: 'Component Detection - OSS Compliance'
+  inputs:
+    ignoreDirectories: '$(Build.SourcesDirectory)/tests/SampleApps'
```
Comment (william-msft) [inline vsts/pipelines/autoUpdater.yml:16]: @arroyc should we add `ignoreDirectories: '$(Build.SourcesDirectory)/tests/Oryx.Integration.Tests/bin/Debug/SampleApps'`? I see this path triggering alerts too: https://devdiv.visualstudio.com/DevDiv/_componentGovernance/107100/alert/3859190?typeId=8187690

Comment (arroyc) [reply vsts/pipelines/autoUpdater.yml:16]: thanks you noticed this ... as discussed over teams channel .. I'm going to ignore the test directory itself

**Review by qianz2 — APPROVED** [review]

**Review by gpcastro — APPROVED** [review]

**Review by william-msft — APPROVED** [review]

**Pull request merged**